### ISSUE TITLE
clientSpecified option does not render widget

### ISSUE THREAD

**Issue by ruhusobeje** [zouhir/preact-habitat#23]
@zouhir 
I tried the login example under this repo.
But, however we try using clientSpecified, it does not render the plugin.

using data-mount-in and:
`niceLogin.render({
    clientSpecified: true,
    inline: false,
    clean: false
  });`

Is it released as stated in the readme or is it under another branch? 🤔 

PS: Btw, Zouhir thanks for Preact habitat. Works great for all our purposes at work❤️

**Comment by zouhir** [comment]
I will look at this issue for you! sounds like a 🐜 somehwere!

also Thank you for the awesome message 😍😍😍😍😍😍😍

**Comment by felipewer** [comment]
@ruhusobeje 
I created PR #28 to fix this, but while it is not released there is a workaround (at least for v3.2.1):

In your client html swap the **class** attribute for the **data-widget** attribute and make sure the value is exactly the same as in the **data-mount-in** attribute.
```
<div data-widget="my-awesome-widget"></div>
<script async src="./src/index.js" data-mount-in="my-awesome-widget"></script>
```

**Comment by zouhir** [comment]
fixed in 3.2.2
